src/app/containers/ATIAnalytics/params/article/index.test.jsx

```diff
@@ -48,80 +58,27 @@ describe('ArticleAtiParams', () => {
     previousPath: '/previous-path',
     service: 'news',
     statusCode: 200,
+    atiAnalyticsAppName: 'news',
+    atiAnalyticsProducerId: 0,
   };
 
   describe('atiPageViewParams is called ', () => {
     afterEach(() => {
       jest.clearAllMocks();
     });
 
-    it('should call atiPageViewParams with the params from the Contexts', () => {
-      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
-      atiPageViewParams.default = mock;
-
-      renderer.create(Component(newsServiceContextStub, requestContextStub));
-
-      expect(mock).toHaveBeenCalledTimes(1);
-      expect(mock).toHaveBeenCalledWith({
-        appName: 'news',
-        contentId: 'urn:bbc:optimo:c0000000000o',
-        contentType: 'article',
-        isUK: true,
-        language: 'language',
-        ldpThingIds: 'id',
-        ldpThingLabels: 'label',
-        pageIdentifier: 'news.articles.c0000000000o.page',
-        pageTitle: 'A headline',
-        producerId: 0,
-        platform: 'canonical',
-        service: 'news',
-        statsDestination: 'NEWS_PS_TEST',
-        timePublished: '2000-01-01T00:00:00.000Z',
-        timeUpdated: '2001-01-01T00:00:00.000Z',
-        previousPath: '/previous-path',
-        origin: 'https://www.test.bbc.co.uk',
-      });
-    });
-
-    it('should call article utility functions with arguments', () => {
-      testUtils.getContentId = jest.fn();
-      testUtils.getLanguage = jest.fn();
-      testUtils.getPageIdentifier = jest.fn();
-      testUtils.getPromoHeadline = jest.fn();
-      testUtils.getThingAttributes = jest.fn();
-      commonTestUtils.getPublishedDatetime = jest.fn();
-
-      renderer.create(Component(newsServiceContextStub, requestContextStub));
-
-      expect(testUtils.getContentId).toHaveBeenCalledTimes(1);
-      expect(testUtils.getContentId).toHaveBeenCalledWith(mockArticleData);
-      expect(testUtils.getLanguage).toHaveBeenCalledTimes(1);
-      expect(testUtils.getLanguage).toHaveBeenCalledWith(mockArticleData);
-      expect(testUtils.getThingAttributes).toHaveBeenCalledTimes(2);
-      expect(testUtils.getThingAttributes).toHaveBeenCalledWith(
-        'thingId',
-        mockArticleData,
-      );
-      expect(testUtils.getThingAttributes).toHaveBeenCalledWith(
-        'thingLabel',
-        mockArticleData,
+    it('should call buildArticleATIUrl exactly once', () => {
+      const output = renderer.create(
```

Comment:
for snapshots we prefer using the `@bbc/psammead-test-helpers` package `shouldMatchSnapshot` function.

Reply:
ShouldMatchSnapshot from the `@bbc/psammead-test-helpers` package on this component, gives `<div />` as its output, rather than the wholesome
```
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {
    "content": Object {},
    "metadata": Object {...},
      "passport": Object {...},
      "tags": Object {...},
    },
    "promo": Object {... },
  },
  "rendered": null,
  "type": [Function],
}
```

Reply:
I think the object you referenced there is not the output but is actually the input. Why does `ArticleAtiParams` not render anything? What does it do? Is it even a React component?

Reply:
I've taken out 

- params/article/index.test.jsx
- params/frontpage/index.test.jsx
- params/media/index.test.jsx

and their test + snapshot files.

These files were leftovers from the `latest` branch, and it turns out we don't need them anymore, since `ATIAnalytics/index.jsx` can depend on `buildParams.js` files directly, which contain regular javascript functions, and make testing easy.